### ISSUE TITLE
Play! Java project compatibility/example?

### ISSUE THREAD

**Issue by arashi01** [teamon/play-navigator#1]
First off just wanted to thank you for the great work!

Just checking if this is usable with a Play! 2.0 Java project; and if so, if you know of any examples available online?

Cheers!

**Comment by teamon** [comment]
Thanks!

In Play 2.0 you can mix scala and java.
To make routes definitions look like they look play-navigator uses a
bit advanced scala features (you can read more about it here
http://codetunes.com/2012/05/09/scala-dsl-tutorial-writing-web-framework-router
)
On the other hand, calling e.g. nav.home() would look something like
nav.home.apply(scala.None$.apply())
since you would have to provide all arguments (in scala there are
default arguments values).
To sum up, you _could_ use it, but it won't be as nice as in scala.

Cheers!

(learn scala, it is a great language!)

Wysłane z iPhone'a

Dnia 18 lip 2012 o godz. 05:17 arashi01
reply@reply.github.com
napisał(a):

**Comment by arashi01** [comment]
Thanks. Will give Scala a go!

Regards,
Ali
